doc/toolbox.1.md

```diff
@@ -42,7 +42,11 @@ Print a synopsis of this manual and exit.
 
 **--verbose, -v**
 
-Print debug information including standard error stream of internal commands.
+Print toolbox debug information including standard error stream of internal commands.
+
+**--very-verbose, -vv**
```

Comment:
This wording seems a bit awkward and I can't think of any better way to phrase it. eg., `-vv` might affect more than just `podman` in the future.

Moreover, we want to advertise `-vv` as the mechanism to increase the log level. The fact that it's implemented as a separate `--very-verbose` instead of `--verbose --verbose` is an unimportant detail that's a consequence of various technical decisions.

Once we have better command line parsing facilities (in the rewritten Go version), I'd rather not have a separate `--very-verbose` any more.

Given all that, it seems easier to say something like *use -vv for more*.